tools/release.sh

```diff
@@ -28,11 +28,21 @@ if [ "$PUBLIC" = "public" ] && ! [ -f ../ankidroiddocs/changelog.asc ]; then
   echo "Could not find ../ankidroiddocs/changelog.asc?"
   exit 1
 fi
+if [ "$PUBLIC" = "public" ] && ! [ -f ../ankidroiddocs/manual.asc ]; then
+  echo "Could not find ../ankidroiddocs/manual.asc?"
+  exit 1
+fi
+if [ "$PUBLIC" = "public" ] && ! [ -f ../ankidroiddocs/help.asc ]; then
+  echo "Could not find ../ankidroiddocs/help.asc?"
+  exit 1
+fi
```

Comment:
I agree with David here - the changelog does not make sense in non-public releases because we just don't update it, but the help and manual could see updates we actually want users to see + verify so they should be generated all the time

these code suggestions are untested

```suggestion
if ! [ -f ../ankidroiddocs/manual.asc ]; then
  echo "Could not find ../ankidroiddocs/manual.asc?"
  exit 1
fi
if ! [ -f ../ankidroiddocs/help.asc ]; then
  echo "Could not find ../ankidroiddocs/help.asc?"
  exit 1
fi
```